docs/installation.md

```diff
@@ -53,7 +53,7 @@ Information about the binaries in the release tarball:
 
 |           Binary Name          |      Support      |   OS  | Architecture |
 |:------------------------------:|:-----------------:|:-----:|:------------:|
-|          cri-containerd        | seccomp, apparmor | linux |     amd64    |
+|            containerd          | seccomp, apparmor | linux |     amd64    |
```

Comment:
Remove this line, and put `seccomp, apparmor` into containerd row.

Reply:
that's what that edit did..

Reply:
@mikebrow There is another `containerd/containerd-shim` row below.

Reply:
done

Reply:
add `overlay, btrfs` for containerd.

Reply:
done